src/utils/fetchJson.ts

```diff
@@ -1,6 +1,6 @@
 import { HttpVerbs } from "src/@types/HttpVerbs";
 
-type Method = Exclude<HttpVerbs, "GET">;
+type Method = Exclude<HttpVerbs, HttpVerbs.GET>;
```

Comment:
I'd say `Methods` would precise that there are more than one. Also we may find a better wording, maybe `BodyMethods`? @Fenntasy Not sure.

Reply:
OK, for now it's changed for Methods 2d71ba8

Reply:
Well I'd stick with Method as we type the value `fetchJson` is expecting and it only expects one at a time.
It could be `HttpVerb` or `HttpMethod` but it's not linked to the body per se (the fact that there is a body is because of the method but the reverse is not true)

Reply:
Make sense. What about `Method` vs `Methods`?

Reply:
Well let's reverse the thinking, you want to use `fetchJson` on VSCode, you type `fetchJson(` what do you expect to see in the argument lists? 

I would expect that the function wants a `Method`, wouldn't you?

Reply:
I'd want to pick a method from a pool of method.

Reply:
yes but `Method` can be `POST | PUT | PATCH | DELETE` 😅

Reply:
Then, you can revert to `Method` hamza, sorry 🙇.